pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
Hi, Thanks for the code.  Looks like a mismatch with 'y' and 'z'.  Let us know how you get on.